vic/drivers/shared_image/src/vic_store.c

```diff
@@ -25,12 +25,14 @@
  *****************************************************************************/
 
 #include <vic_driver_shared_image.h>
+#include <vic_driver_cesm.h>
```

Comment:
this file is also compiled with the image driver so you need to find a solution that does not require the cesm driver.  I think that all you need is the caseid which can be stored as `filenames.statefile`.